src/templates/ConfigSetting.ts

```diff
@@ -0,0 +1,70 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ConfigValidator } from './ConfigValidator';
+import { ConfigVariables } from './ConfigVariables';
+
+export enum ValueType {
+    string = 'string',
+    boolean = 'boolean',
+    enum = 'enum',
+    checkBoxList = 'checkBoxList'
+}
+
+export enum ResourceType {
+    DocumentDB = 'DocumentDB'
+}
+
+interface IBindingSetting {
+    name: string;
+    value: ValueType;
+    label: string;
+    defaultValue?: string;
```

Comment:
defaultValue is a property name, not an actual value?   If so, should be documented

Reply:
Or better name.

Reply:
It is a value

---
In reply to: [147804409](https://github.com/Microsoft/vscode-azurefunctions/pull/32#discussion_r147804409) [](ancestors = 147804409)